src/particles/plasma/PlasmaParticleContainerInit.cpp

```diff
@@ -38,10 +38,28 @@ InitParticles (const amrex::IntVect& a_num_particles_per_cell,
     const amrex::Real scale_fac = Hipace::m_normalized_units?
                                   1./num_ppc : dx[0]*dx[1]*dx[2]/num_ppc;
 
+
+    amrex::IntVect box_nodal{0, 0, 0};
+    amrex::IntVect box_grow{0, 0, 0};
+    amrex::Real x_offset = 0._rt;
+    amrex::Real y_offset = 0._rt;
```

Comment:
Just in case, there is built-in logic in AMReX to do this kind of things, see e.g. `amrex::IndexType::NODE`. Might help a bit or not.

Reply:
Not much